kview/src/main/java/dev/ikm/komet/kview/controls/skin/KLComponentListControlSkin.java

```diff
@@ -306,7 +323,21 @@ protected void layoutChildren(double contentX, double contentY, double contentWi
         double x = contentX + padding.getLeft();
         double y = contentY + padding.getTop();
         titleLabel.resizeRelocate(x, y, labelPrefWidth, labelPrefHeight);
```

Comment:
This isn't part of your PR and it's a small nitpick but can you rename labelPrefWidth and labelPrefHeight to titlePrefWidth and titlePrefHeight?